libs/agent-scripts/cloudbroker/upgrades/2.4.4/master/migrate_ros_config.py

```diff
@@ -22,7 +22,10 @@ def action():
     args['script'] = '/system hardware set multi-cpu=no'
     for vfw in vfws:
         args['fwobject'] = vfw
-        acl.executeJumpscript('jumpscale', 'vfs_runscript_routeros', args=args, nid=vfw['nid'], gid=vfw['gid'], timeout=5)
+        try:
+            acl.executeJumpscript('jumpscale', 'vfs_runscript_routeros', args=args, nid=vfw['nid'], gid=vfw['gid'], timeout=5)
+        except:
+            j.console.warning("Can't connect to routeros {}".format(vfw['guid']))
```

Comment:
replace console warning with `j.errorconditionhandler.raiseOperationalWarning` so it get's logged into the system